docs/sources/getting-started/getting-started-prometheus.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Getting started with Grafana and Prometheus"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+[menu.docs]
+identifier = "getting_started-grafana-prometheus"
+parent = "guides"
+weight = 300
++++
+
+# Getting Started with Grafana and Prometheus
+
+Prometheus is an open-source systems monitoring and alerting toolkit for which Grafana includes built-in support. This topic walks you through the steps to create a series of dashboards in Grafana displaying system metrics of a host monitored by Prometheus.
+
+You can also configure your Grafana instance in the cloud to display system metrics. For more information, refer to the [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) documentation.
+
+## Step 1: Install Grafana and build your first dashboard
+
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+- Install Grafana
+- Login to Grafana
+- Create your first dashboard
+
+## Step 2: Download Prometheus and node exporter
+
+Prometheus, like Grafana, can be installed on many different operating systems. Refer to the [Prometheus download page](https://prometheus.io/download/) which lists all stable versions of Prometheus components. Download the following components:
+ - [prometheus](https://prometheus.io/download/#prometheus)
+ - [node_exporter](https://prometheus.io/download/#node_exporter)
+
+## Step 3: Install Prometheus node_exporter
+
+Prometheus Node Exporter is a widely used tool that exposes system metrics. Install node_exporter on all hosts you want to monitor. For instructions on how to install node_exporter, refer to the [Installing and running the Node Exporter](https://prometheus.io/docs/guides/node-exporter/#installing-and-running-the-node-exporter) section in Prometheus documentation.
```

Comment:
Nit: Three different spellings of node exporter so far:
- "node exporter"
- "Node Exporter"
- "node_exporter"

I'd say each is fine, but we should try to use one spelling consistently.

Reply:
Good catch! We will go with "node_exporter".

Reply:
@achatterjee-grafana , please add this to the Word usage section of the Grafana style guide.